R/plot.R

```diff
@@ -0,0 +1,323 @@
+#' @title Static rasterly graphics
+#' @description Render a \code{rasterly} object. 
+#' This object can involve axes, legend, etc to better convert graphics to data.
+#' @name static
+#' @param rasterlyObj A \code{rasterly} object.
+#' @param xlim The x limits (x1, x2) of the plot.
+#' @param ylim The y limits (x1, x2) of the plot.
+#' @param xlab A y label for the plot.
+#' @param ylab A x label for the plot.
+#' @param main A title for the plot.
+#' @param sub A sub title for the plot.
+#' @param interpolate A logical value indicating whether to linearly interpolate the image
+#' (the alternative is to use nearest-neighbour interpolation, which gives a more blocky result).
+#' @param axes Show axes or not.
+#' @param legend Show legend or not.
+#' @param legendlabel The label of legend.
+#' @param legendlayer The legend represents which layer in \code{rasterly} object. Default is 1.
+#' @param legendmain The main title of legend
```

Comment:
@z267xu Would call this `legend_main` to match syntax of other args.

```suggestion
#' @param legendmain Character. The main title to use within the figure legend. The default is `NULL`, which omits the figure legend title.
```

Reply:
👍 💯